kotlin/internal/jvm/android.bzl

```diff
@@ -76,6 +76,7 @@ def _kt_android_artifact(
             enable_data_binding = enable_data_binding,
             tags = tags,
             visibility = ["//visibility:private"],
+            **kwargs
```

Comment:
@oliviernotteghem is there a reason you opted to not pass the full kwargs for this case?

Reply:
@pswaminathan I think you also want to check for assets in the conditional that gates this logic.

```
if resource_files or manifest or assets:
```